packages/frontend2/src/app/(legacy)/scaling-next/projects/[slug]/page.tsx

```diff
@@ -0,0 +1,82 @@
+import { layer2s, layer3s } from '@l2beat/config'
+import { assert } from '@l2beat/shared-pure'
+import { notFound } from 'next/navigation'
+import { HighlightableLinkContextProvider } from '~/app/_components/link/highlightable/highlightable-link-context'
+import { ProjectDetails } from '~/app/_components/projects/project-details'
+import { DesktopProjectNavigation } from '~/app/_components/projects/sections/navigation/desktop-project-navigation'
+import { MobileProjectNavigation } from '~/app/_components/projects/sections/navigation/mobile-project-navigation'
+import { getScalingProjectEntry } from '~/server/features/scaling/project/get-scaling-project-entry'
+import { getDefaultMetadata } from '~/utils/get-default-metadata'
+import { ScalingProjectSummary } from './_components/scaling-project-summary'
+
+const scalingProjects = [...layer2s, ...layer3s]
+
+export async function generateStaticParams() {
+  return scalingProjects.map((layer) => ({
+    slug: layer.display.slug,
+  }))
+}
+
+export async function generateMetadata({ params }: Props) {
+  const project = scalingProjects.find(
+    (layer) => layer.display.slug === params.slug,
+  )
+  assert(project, 'Project not found')
```

Comment:
I'd consider replacing with `notFound()`, even if we're generating static params.